data/monitoring/prom-rule.yaml

```diff
@@ -11,6 +11,9 @@ spec:
       rules:
         - expr: sum(up{namespace='{{ .Namespace }}', pod=~'cluster-network-addons-operator-.*'} or vector(0))
           record: kubevirt_cnao_num_up_operators
+          annotations:
+            summary: Total count of running CNAO operators
```

Comment:
Similarly, I'm not sure if using labels is the right thing for documentation strings.
IIUC, labels should be values of similar meaning, that one may use for filtering.
Surely, nobody will ever filter the record by the content of summary.